server.js

```diff
@@ -4,22 +4,475 @@ const path = require('path');
+                                    'Результаты свидетельствуют о контролируемости, социальной точности, следовании «Я»-образу (высокой интеграции).', 
+                                    'Такие результаты свидетельствуют об организованности, умении хорошо контролировать свои эмоции и поведение. Подобные личности способны эффективно управлять своей энергией и умеют хорошо планировать свою жизнь. Они думают, прежде чем действовать, упорно преодолевают препятствия, не останавливаются при столкновении с трудными проблемами, склонны доводить начатое до конца и не дают обещания, которые не могут выполнить. Подобные люди хорошо осознают социальные требования и заботятся о своей общественной репутации.'
+                                ],
+                        clar: '',
+                        stan: 0
+                    },
+                    {
+                        name: 'Напряженность (Фактор Q4)',
+                        stan_bar: [0, 5, 7, 9,  12, 15, 18, 20, 22, 24, 26],
+                        options: [
+                                    'Такие результаты характерны для людей, которые отличаются расслабленностью, отсутствием сильных побуждений и желаний. Они невозмутимы, спокойно относятся к неудачам и удачам, находят удовлетворение в любом положении дел и не стремятся к достижениям и переменам.', 
+                                    'Результаты свидетельствуют о расслабленности (ненапряженности), нефрустрированности.', 
+                                    'Результаты свидетельствуют о напряженности, фрустрированности, побуждаемости, сверхреактивности (высоком энергетическом напряжении).', 
+                                    'Такие результаты характерны для человека, у которого выражен классический невроз тревожности. Подобные люди постоянно находятся в состоянии возбуждения, с большим трудом успокаиваются, чувствуют себя разбитыми, усталыми, и не могут оставаться без дела даже в обстановке, способствующей отдыху. Для таких людей характерны эмоциональная неустойчивость с преобладанием пониженного настроения, раздражительность, проблемы со сном, негативное отношение к критике.'
+                                ],
+                        clar: '',
+                        stan: 0
+                    }
+                ];
+
+                for (let i = 0; i < 16; i++) {
+                    for (let j = 1; j < 11; j++) {
+                        if (results[i].Value == 0) {
+                            kettell_processing_params[i].stan = 1;
+                        }
+                        else if ((results[i].Value > kettell_processing_params[i].stan_bar[j-1]) && (results[i].Value <= kettell_processing_params[i].stan_bar[j])) {
+                            kettell_processing_params[i].stan = j;
+                        }
+                    }
+                }
+
+                for (let i = 0; i < 16; i++) {
+                    if (kettell_processing_params[i].stan <= 3 && kettell_processing_params[i].stan >= 1) {
```

Comment:
I would swap it for better readability, so mix first, max last:
```suggestion
                    if (kettell_processing_params[i].stan >= 1 && kettell_processing_params[i].stan <= 3) {
```